viadot/flows/sap_rfc_to_adls.py

```diff
@@ -27,6 +27,8 @@ def __init__(
         overwrite: bool = False,
         adls_sp_credentials_secret: str = None,
         vault_name: str = None,
+        update_kv: bool = False,
+        field_to_refresh: str = None,
```

Comment:
Maybe it will be better to put a more meaningful name such as filter_column or filter_value_column